docs/content/docs/testing/telemetry.md

```diff
@@ -13,21 +13,21 @@ your other telemetry, and route the event data in whatever way you need.
 ## Collect Fluid Framework logs with a custom `ITelemetryBaseLogger`
 
 The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can
-be implemented and passed into the client's `createContainer()` and `getContainer()` methods via the config parameter.
+be implemented and passed into the client's constructor via the props parameter.
 
-All Fluid service clients (for example, `AzureContainerConfig` and `TinyliciousContainerConfig`) allow passing a `logger?:
-ITelemetryBaseLogger` into the service client configuration. Both `createContainer()` and `getContainer()` methods will
-create an instance of a `Loader` class object, where the logger defined in the service client configuration is passed in
+All Fluid service clients (for example, `AzureClientProps` and `TinyliciousClientProps`) allow passing a `logger?: ITelemetryBaseLogger`
+into the service client props. Both `createContainer()` and `getContainer()` methods will
+create an instance of a `Loader` class object, where the logger defined in the service client props is passed in
 as an optional parameter, `ILoaderProps.logger`, in the `Loader` constructor argument.
 
-[TinyliciousContainerConfig](https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/tinylicious-client/src/interfaces.ts#L9)
-interface definition takes an optional parameter `logger`. (The definition is similar to `AzureContainerConfig` interface)
+[TinyliciousClientProps](https://github.com/microsoft/FluidFramework/blob/main/packages/framework/tinylicious-client/src/interfaces.ts#L17)
```

Comment:
linking to the code like this doesn't seem maintainable.  is this definition in the doc site api reference somewhere?  it would be better to link there instead if you can because it's more stable, for this and elsewhere

Reply:
I couldn't find anything on the official website [here](https://fluidframework.com/docs/testing/tinylicious/), I think it is yet to be updated or written. The closest thing I found that mentioned about the props is [here](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/tinylicious-client#instantiating-tinyliciousclient)

Reply:
We should link to the package docs, but they're not published right now. Once #7472 is decided, we can add appropriate links. I recommend removing the links for now and following up to add them later.

Reply:
Right, I will create a follow-up Github issue for this. 😊

Reply:
Github issue created #7480